compiler_opt/rl/generate_bc_trajectories.py

```diff
@@ -0,0 +1,92 @@
+# coding=utf-8
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module for running compilation and collect data for behavior cloning."""
```

Comment:
nit: could "behavior cloning" be confused with the straight-forward, "try to do exactly like this policy" one? do you want to use "imitation learning"?

Reply:
...or whatever the appropriate name is - my main concern is confusion with the use "bc" elsewhere in this codebase. If the right approach would be to rename those other places, sgtm (as in, we capture that in an issue and somebody will pick it up at a point).

Reply:
Behavior cloning in this context exactly refers to once we collect the set of trajectories that we like, e.g., best trajectory per module, then we want to behave exactly as these trajectories prescribe. I think it's still appropriate to use behavior cloning here.

Reply:
sgtm!